docs/DeveloperGuide.md

```diff
@@ -314,6 +314,24 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
+**Use case: Install the app**
+MSS 
+1. User execute the installation package
+2. The app ask for continue or cancel with a pop up window
+3. User click continue
+4. The app install the app
+5. The app show a pop up window to inform user the installation is successful
+Use Case ends
+
+Extensions
+* 3a. User click cancel
+  * 3a1. The app close the installation package
```

Comment:
Same as above, perhaps the extension could be just: User cancels the Installation

Reply:
sure, will fix that